### ISSUE TITLE
Zoneless angular support

### ISSUE THREAD

**Issue by banned007** [tiberiuzuld/angular-gridster2#916]
Please add support for zoneless angular, maybe a new package/repo for this? not sure how to proceed, I have a working version of zoneless (literlly just removed the calls of zone functions, and it seemed to work right out) on my fork, but not sure if this is something you should include in the main package or if you want to create a new one for supporting this feature.

**Comment by banned007** [comment]
here is my fork: https://github.com/banned007/angular-gridster-zoneless

**Comment by tiberiuzuld** [comment]
Hello,
Thank you for trying this. Was also curious if it will work.
I normally wait till the feature is stable in angular before integrating in the library.
Will see if it's possible to have it as a config option or make a branch with a separate version in npm.

**Comment by banned007** [comment]
Hmm I don't think a config option is feasible, as you would need to remove zone.js from the peerDeps, and when you inject it, you might get some injection dependency issues, so I think the most straight forward approach would be to just publish a new package. 
also maybe having signals in the html would eliminate the need for having the ChangeDetectionRef.markForCheck call, but this could be implemented in a subsequent version of zoneless?

**Comment by tiberiuzuld** [comment]
Yes you are right, the most viable solution I see is to make a branch with the changes and make a version with boneless and signals, something like 18.0.0-z to be published on npm.

Will see when I would have the time for this, most probably next week.

**Comment by tiberiuzuld** [comment]
New version 18.0.0-z is on npm https://www.npmjs.com/package/angular-gridster2/v/18.0.0-z
Will add signals and remove the change detection at a later time.

**Comment by frellan** [comment]
Zoneless is now stable. https://angular.dev/guide/zoneless
